ecl/eclagent/eclgraph.cpp

```diff
@@ -853,9 +853,16 @@ void EclSubGraph::updateProgress()
 {
     if (!isChildGraph && agent->queryRemoteWorkunit())
     {
-        Owned<IWUGraphStats> progress = parent.updateStats(queryStatisticsComponentType(), queryStatisticsComponentName(), id);
+        Owned<IWUGraphStats> progress = parent.updateStats(queryStatisticsComponentType(), queryStatisticsComponentName(), agent->getWorkflowId(), id);
         IStatisticGatherer & stats = progress->queryStatsBuilder();
         updateProgress(stats);
+
+        StringBuffer subgraphid;
+        subgraphid.append(parent.queryGraphName()).append(":").append(SubGraphScopePrefix).append(id);
+        if (startGraphTime)
+            parent.updateWUStatistic(SSTsubgraph, subgraphid, StWhenStarted, "", startGraphTime);
```

Comment:
Should pass nullptr as the description rather than ""

Reply:
done